iscsi/multipath.go

```diff
@@ -108,14 +109,21 @@ func RemoveAndClear(device string) error {
 	debug.Printf("Remove and clear multipath device (%s)\n", device)
 
 	// Remove device-mapper logical device
+	debug.Printf("dmsetup remove -f %s\n", device)
 	if output, err := execCommand("dmsetup", "remove", "-f", device).CombinedOutput(); err != nil {
+		exitErr, ok := err.(*exec.ExitError)
+		if ok && exitErr.ProcessState.Sys().(syscall.WaitStatus).ExitStatus() != 0 {
```

Comment:
Is this equivalent to "exitErr.ExitCode()" or is it doing something different with the wait status? 
https://pkg.go.dev/os#ProcessState.ExitCode

Reply:
Yes, it is, and that is much clearer. I will make the change, thank you.